components/function-controller/pkg/apis/serverless/v1alpha2/function_types.go

```diff
@@ -48,7 +48,7 @@ type Source struct {
 type InlineSource struct {
 	Source string `json:"source"`
 	//+optional
-	Dependencies string `json:"dependencies"`
+	Dependencies string `json:"dependencies,omitempty"`
```

Comment:
Why are we have still string type here? If you would look [here](https://github.com/kyma-project/kyma/pull/15307/files#diff-baa692970ec321b9fde546a7f3a0a0e237852767415d15fdd58a469e1847d70bR135) we have a pointer to string. What is the difference?

Reply:
The pointer was reverted